compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -11,6 +11,8 @@ class JSExportTest extends DirectTest with TestHelpers {
 
   @Test
   def noDoubleUnderscoreExport = {
+    // scalastyle:off
```

Comment:
It's quite annoying indeed. But if you miss turning `scalastyle:on` then all the code after that (may even spill to the next files) won't get check until it is turned on again.